backend/src/packages/chaiNNer_pytorch/settings.py

```diff
@@ -66,12 +66,24 @@
     ),
 )
 
+package.add_setting(
+    NumberSetting(
+        label="Memory Budget Limit (GiB)",
+        key="budget_limit",
```

Comment:
A budget limit for what?

Reply:
This is the same name as the NCNN setting, for consistency